mkt/extensions/indexers.py

```diff
@@ -78,6 +78,12 @@ def get_mapping(cls):
             }
         }
 
+        # Attach boost field, because we are going to need search by relevancy.
+        cls.attach_boost_mapping(mapping)
```

Comment:
Yep, it's in the base Indexer class. Re-used in webapps, websites and extensions.